cdk/README.md

```diff
@@ -0,0 +1,23 @@
+# Infrastructure
+
+This directory defines the components to be deployed to AWS
+
+## Useful commands
+
+We follow the [`script/task`](https://github.com/github/scripts-to-rule-them-all) pattern,
+find useful scripts within the [`script`](./script) directory for common tasks.
+
+- `./script/setup` to install dependencies
+- `./script/start` to run the Jest unit tests in watch mode
+- `./script/lint` to lint the code using ESLint
+- `./script/test` to lint, run tests and generate templates of the CDK stacks
+- `./script/diff` to print the diff between a CDK stack and a running CloudFormation stack
```

Comment:
Shall we remove these `script` files? (Since [GuCDK 43.3.0](https://github.com/guardian/cdk/releases/tag/v43.3.0), they are no longer created too.)

Reply:
https://github.com/guardian/mobile-save-for-later/pull/64/commits/76a530ac3054453d03cc18dca6a44f1f5f4c4896